src/gax/Cargo.toml

```diff
@@ -28,9 +28,9 @@ repository.workspace = true
 [dependencies]
 base64      = "0.22"
 bytes       = "1"
-futures     = { version = "0.3", optional = true }
+futures     = { version = "0.3" }
```

Comment:
I finally understand what you did here: you are using streams in the implementation, and hiding the stream trait implementation under a feature.

That works, but it may be possible to improve the implementation to remove this dependency.  For example, the pollers do this:

https://github.com/googleapis/google-cloud-rust/blob/ff3abcc1b5753e53e9016718fbc27f2555f32eee/src/lro/src/lib.rs#L279-L295

Reply:
I thought unstable crates in the implementation were fair game. We also have `base64` and `rand` which are < 1.0. I take it you are more scared of `futures` breaking things?

FWIW: I originally went down the other path where the dep on `futures` is feature-gated behind `unstable-stream`. It would look something like this:

https://github.com/dbolduc/google-cloud-rust/blob/5d07ad8a3e6cef2bfc48934a8cfcf4b0f95df010/src/gax/src/paginator.rs#L41-L103

But then I realized I could save myself a ton of trouble by using streams in the implementation.